src/Microsoft.ApplicationInsights/Extensibility/Implementation/Experimental/ExperimentalFeatures.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Experimental
+{
+    /// <summary>
+    /// This class provides a means to interact with the <see cref="TelemetryConfiguration.ExperimentalFeatures" />.
+    /// This performs a simple boolean evaluation; does a feature name exist in the string array?
+    /// Evaluation results are cached.
+    /// </summary>
+    internal static class ExperimentalFeatures
+    {
+        internal static bool IsExampleFeatureEnabled(TelemetryConfiguration telemetryConfiguration) => telemetryConfiguration.EvaluateExperimentalFeature("exampleFeature");
```

Comment:
is this API needed?

Reply:
I'm generally in favor of apis that wrap string values.
I think we should have class that lists all the expected experimental features.
I admit this wrapper class made more sense when it also included the cache variable.

**Would you prefer class of string constants?**

Reply:
since this is just an example and does not ship within base SDK, I propose to decide on case-by-case basis when we ship actual features in the web/aspnetcore/logging